### ISSUE TITLE
isoAnnotLite output filename is not correct, leads to apparent missing output

### ISSUE THREAD

**Issue by joelnitta** [ConesaLab/SQANTI3#155]
When running `sqanti3_qc.py` with `--isoAnnotLite`, the path for the output `gff3` is missing a slash, which leads to the file being output to an unexpected place so it may appear missing to the user.

For example, with the command (in particular notice `-d /tmp/analysis`)

```
sqanti3_qc.py d_magna.collapsed.filtered.gff daphnia_genome.gtf daphnia_genome.fna -d /tmp/analysis -o d_magna --report pdf --isoAnnotLite
```

results in the following (output truncated to only the relevant part)

```
SQANTI3 complete in 929.2940882518888 sec.


Running IsoAnnot Lite 2.7.3 (using all gene information) ...

Reading SQANTI 3 Files and creating an auxiliar GFF...
Adding extra information to relative columns...
Reading GFF3 to sort it correctly...
Generating final GFF3...
Time used to generate new GFF3: 9.81 seconds.

Exportation complete.
Your GFF3 result is: '/tmp/analysisd_magna.gff3'
```

The `gff3` file is `/tmp/analysisd_magna.gff3`, not `/tmp/analysis/d_magna.gff3`.

Thus the file does not get written to the expected location, and may appear missing to the user.

sqanti3 v5.0 (docker image: joelnitta/sqanti3:5.0 | https://github.com/joelnitta/sqanti3-docker | https://hub.docker.com/r/joelnitta/sqanti3/tags)

**Comment by aarzalluz** [comment]
Hi @joelnitta,

Thank you for reporting this! I'm taking note so that we can fix this in the next release.

Best,

Ángeles

**Comment by aarzalluz** [comment]
Fixed in v5.1
